core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -435,17 +437,24 @@ private void createMetadataTableIfNotExists() throws ExecutionException {
                       .readCapacityUnits(METADATA_TABLE_REQUEST_UNIT)
                       .writeCapacityUnits(METADATA_TABLE_REQUEST_UNIT)
                       .build())
-              .tableName(getMetadataTable())
+              .tableName(getFullTableName(metadataNamespace, METADATA_TABLE))
               .build());
     } catch (Exception e) {
       throw new ExecutionException("creating the metadata table failed", e);
     }
-    waitForTableCreation(getMetadataTable());
+    waitForTableCreation(metadataNamespace, METADATA_TABLE);
+
+    if (!noBackup) {
+      enableContinuousBackup(metadataNamespace, METADATA_TABLE);
+    }
```

Comment:
This enables PITR for the metadata table. It uses the existing method `enableContinuousBackup()` to enable PITR.